### PR TITLE
OrganizationValue.java out of memory during compilation fix

### PR DESCRIPTION
**Pull request by amallemadugula1**
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE0000189LjjYAE/view

Updated crossinstance.wsdl cannot be checked in because resulting wsc stub OrganizationValue.java cannot be compiled due to too many enum values exceeding the byte limit for the java compiler. 

This fixes it by removing removing the valuesToEnums field in the enum. This field was added [here](https://github.com/forcedotcom/wsc/commit/10e5d914cee066a4927e1e5f641b5fa8ca528687) long ago to allow for hyphens. This functionality will now be removed only for OrganizationValues.java. This brings it under the byte limit.

This is only hard coded for OrganizationValues.java. The rest of the generated files should remain the same. Eventually a solution should be found such that this hard coded exception doesn't have to exist.

### CODE REVIEW

**Review by lzemskov — APPROVED** [review]

**Review by abhinavChadda — APPROVED** [review]

File: src/main/java/com/sforce/ws/codegen/templates/simpleTypeOrganizationValue.st [hunk 1]
```diff
@@ -0,0 +1,25 @@
+simpleTypeOrganizationValue(gen) ::= <<
+package $gen.packageName$;
+
+import java.util.EnumSet;
```
Comment (lzemskov) [inline src/main/java/com/sforce/ws/codegen/templates/simpleTypeOrganizationValue.st:4]: do you need all these imports?

File: src/main/java/com/sforce/ws/codegen/templates/simpleTypeOrganizationValue.st [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This is a generated class for the SObject Enterprise API.
+ * Do not edit this file, as your changes will be lost.
+ */
+public enum $gen.className$ {
+
```
Comment (lzemskov) [inline src/main/java/com/sforce/ws/codegen/templates/simpleTypeOrganizationValue.st:13]: nit picks here:
-the comment text in the template does not really add much value so not sure why we need it at all.
-also personally would drop the xtra spaces

File: pom.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
   <groupId>com.force.api</groupId>
   <artifactId>force-wsc</artifactId>
   <packaging>jar</packaging>
-  <version>56.1.0</version>
+  <version>56.2.0</version>
```
Comment (reubencornel) [inline pom.xml:12]: Please revert this change. This should be done by the upgrade and on release.

File: src/main/java/com/sforce/ws/codegen/Generator.java [hunk 4]
```diff
@@ -226,7 +227,13 @@ protected TypeMetadataConstructor newTypeMetadataConstructor(Types types, Schema
 
     protected void generateClassFromSimpleType(Schema schema, SimpleType simpleType, File dir) throws IOException {
         SimpleClassMetadata gen = new SimpleClassMetadata(schema, simpleType, typeMapper);
-        ST template = templates.getInstanceOf(SIMPLE_TYPE);
+        ST template;
+        if (gen.getClassName().equals("OrganizationValue")) {
```
Comment (reubencornel) [inline src/main/java/com/sforce/ws/codegen/Generator.java:231]: I think this should be a map.
